notebooks/mitocheck-map-plotting.py

```diff
@@ -0,0 +1,427 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
+import pathlib
+import warnings
+
+import numpy as np
+import pandas as pd
+import plotly.express as px
+from copairs.map import aggregate
+
+warnings.filterwarnings("ignore")
+
+
+# In[2]:
+
+
+# Directories
+processed_data_dir = pathlib.Path("../data/processed/")
+figures_dir = pathlib.Path("../figures/").resolve(strict=True)
```

Comment:
Not sure if this dir would already exist? Consider writing a line to make the dir as it is an output dir

Reply:
Yes, there's a `.gitkeep` file in the folder.